src/linker/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please exclude this file from the change.... We should really add this to `.gitignore`.